tests/src/com/unciv/dev/FasterUIDevelopment.kt

```diff
@@ -146,23 +147,23 @@ class FontDesktop : FontImplementation {
         // Empty
     }
 
-    override fun getFontSize(): Int {
-        return Fonts.ORIGINAL_FONT_SIZE.toInt()
-    }
+    override fun getFontSize() = Fonts.ORIGINAL_FONT_SIZE.toInt()
 
     override fun getCharPixmap(char: Char): Pixmap {
         var width = metric.charWidth(char)
-        var height = metric.ascent + metric.descent
+        var height = metric.height
         if (width == 0) {
             height = Fonts.ORIGINAL_FONT_SIZE.toInt()
             width = height
         }
+
         val bi = BufferedImage(width, height, BufferedImage.TYPE_4BYTE_ABGR)
         val g = bi.createGraphics()
         g.setRenderingHint(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON)
         g.font = font
         g.color = java.awt.Color.WHITE
-        g.drawString(char.toString(), 0, metric.ascent)
+        g.drawString(char.toString(), 0, metric.leading + metric.ascent)
```

Comment:
Please make leading plus ascent function of metric, like distanceFromTopToBaseline()

This is hard enough to read as it is without having to hold a picture of the various font height parameters in your head